### DISCUSSION TITLE
3 DEC DEV Session

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1502] [General]
When: 3 DEC --- 2000 UTC+8/1300 CET/1200 GMT/0700 EDT

Where: [via Matrix](https://matrix.to/#/#mesa-dev:matrix.org)

Proposed Agenda:

- Permissions
- GH & auto deploys -- discuss / decide, document policy (added by @jackiekazil)
- Pull Requests - Mesa
- Pull Request Mesa - Geo 

@jackiekazil @rht @Corvince @wang-boyu @Tortar @aiwhoo @nunezco2 @EwoutH Please feel free to add or takeaway from this agenda

**Comment by EwoutH** [comment]
I most likely won't be able to join, due to family festivities.

If possible, I would like to get some movement on:

- #1481
- #1479

I also think it could be interesting to take a look at https://github.com/projectmesa/mesa/issues/591 and https://github.com/projectmesa/mesa/issues/719. Both are about the visualizations of networks (nodes and edges), which could be a great addition to Mesa.

**Reply by tpike3** [reply]
@EwoutH  enjoy the festivities and I will discuss

**Comment by wang-boyu** [comment]
Won't be able to join the meeting either - been busy with the assignments at the end of the semester.

Looking forward to the session notes!

**Reply by tpike3** [reply]
Good luck @wang-boyu

**Comment by tpike3** [comment]
### 3 DEC DEV Session Notes: 

**Permissions Policy Discussion:**

- To ensure Mesa resilience @jackiekazil is going to write up Mesa processes for granting and removing permissions to contributors. The goal is to discuss at next Dev Session January 7th. 

**GH and auto-deploy**

- Group concurred to try out releasing Mesa to PYPI with auto-deploy (related to #1479) and see how it works

**Examples**

- @jackiekazil will complete the long discussed move example into the examples repository (#1529, #963)  

**Collection of Agents By Type**

- Group agreed to add filter to collector so users have function to collect agent by type. The view was although not as performant this as acceptable is providing core Mesa this capability.  #1419; #1481; This will be led by @rht 

**Mesa Release 2.0**

- Release in January/Februrary
- Ensure move of Mesa examples to example repo is included
- Include performant enhancements from @Tortar are included
- Include Collection by Agent Type Included

**Reply by tpike3** [reply]
If I missed anything, or you have questions or comments let me know.
